twisted/conch/ssh/keys.py

```diff
@@ -1002,6 +1154,12 @@ def _toString_OPENSSH(self, extra):
 
         @rtype: L{bytes}
         """
+        
+        # No support for EC keys yet.
```

Comment:
If we're going to have a comment like this in the code, it should include a link to the ticket for implementing the functionality in question, rather than just a vague apology :).